airbyte-workers/Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM ${JDK_IMAGE} AS worker
 
 ARG DOCKER_BUILD_ARCH=amd64
 
+# Grab well-known types file
+COPY WellKnownTypes.json /app
```

Comment:
I would suggest moving this COPY to line 23. This would avoid rebuilding the complete image whenever we change this file.

Reply:
done